### PR TITLE
[Quality] Add CI for formatting

### LINKED ISSUE #57: Add code formatting script & Add CI to check code format

### PR DESCRIPTION
**Pull request by zhuohan123**
Completely close #57

### CODE REVIEW

File: vllm/entrypoints/openai/api_server.py [hunk 1]
```diff
@@ -182,7 +183,8 @@ async def create_chat_completion(raw_request: Request):
                                      "logit_bias is not currently supported")
 
     prompt = await get_gen_prompt(request)
-    error_check_ret = await check_length(request, prompt, engine)
+    model_config = await engine.get_model_config()
```
Comment (WoosukKwon) [inline vllm/entrypoints/openai/api_server.py:186]: A dump question: Can we cache `model_config` a head of time and avoid this function call at run time?

File: vllm/entrypoints/openai/api_server.py [hunk 2]
```diff
@@ -95,15 +96,15 @@ async def get_gen_prompt(request) -> str:
...
-        context_len = engine.engine.model_config.hf_config.max_position_embeddings
-    elif hasattr(engine.engine.model_config.hf_config, "seq_length"):
-        context_len = engine.engine.model_config.hf_config.seq_length
+async def check_length(request, prompt, model_config):
+    if hasattr(model_config.hf_config, "max_sequence_length"):
+        context_len = model_config.hf_config.max_sequence_length
+    elif hasattr(model_config.hf_config, "seq_length"):
+        context_len = model_config.hf_config.seq_length
+    elif hasattr(model_config.hf_config, "max_position_embeddings"):
+        context_len = (model_config.hf_config.max_position_embeddings)
```
Comment (WoosukKwon) [inline vllm/entrypoints/openai/api_server.py:105]:
```suggestion
        context_len = model_config.hf_config.max_position_embeddings
```

**Review by WoosukKwon — APPROVED** [review]
LGTM. Thanks! Left some minor comments.

File: vllm/entrypoints/openai/api_server.py (hunk 1 above)
Comment (zhuohan123) [reply vllm/entrypoints/openai/api_server.py:186]: You are right. Let me fix this.

**Pull request merged**